ngraph/frontend/onnx/frontend/CMakeLists.txt

```diff
@@ -25,19 +37,24 @@ if(COMMAND ie_add_vs_version_file)
                            FILEDESCRIPTION "nGraph ONNX frontend library")
 endif()
 
-target_link_libraries(onnx_ngraph_frontend PRIVATE onnx_importer frontend_manager)
+target_link_libraries(onnx_ngraph_frontend PUBLIC ngraph PRIVATE frontend_manager ngraph::builder onnx_common inference_engine_transformations)
 
 target_include_directories(onnx_ngraph_frontend PUBLIC $<BUILD_INTERFACE:${ONNX_FRONTEND_INCLUDE_DIR}>
                                                 $<INSTALL_INTERFACE:${FRONTEND_INSTALL_INCLUDE}>)
-
 target_include_directories(onnx_ngraph_frontend PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src)
 
+target_compile_definitions(onnx_ngraph_frontend PRIVATE ONNX_OPSET_VERSION=${ONNX_OPSET_VERSION})
+if(NGRAPH_USE_PROTOBUF_LITE)
+    target_compile_definitions(onnx_ngraph_frontend PRIVATE NGRAPH_USE_PROTOBUF_LITE)
+endif()
```

Comment:
Can we introduce alias to onnx_importer for backward compatibility?

Reply:
As I know @mateusztabaka (he is OOO today) try to use an alias, but with failure.
In cmake spec we can find:
```ALIAS targets are not mutable, installable or exportable. They are entirely local to the buildsystem description```

Reply:
We need to create alias in `ngraphConfig.cmake`, not in source tree :)